lib/dash/segment_list.js

```diff
@@ -195,7 +195,7 @@ shaka.dash.SegmentList = class {
    * @param {number} periodStart in seconds.
    * @param {?number} periodDuration in seconds.
    * @param {number} startNumber
-   * @param {!Array.<string>} baseUris
+   * @param {function():!Array.<string>} baseUris
```

Comment:
I get confused when baseUris() is a function that returns them and not an array.  (And I've done it to myself in the past.)  Can we please rename these variables to getBaseUris()?

Reply:
+1

Reply:
Done!